fileio/ft_write_data.m

```diff
@@ -875,12 +875,130 @@ function ft_write_data(filename, dat, varargin)
     data.label = hdr.label;
     data.sampleinfo = [1 size(dat,2)];
     
-    % convert the raw data structure to Homer format
-    nirs = fieldtrip2homer(data, 'event', evt);
+    % devide data in nirs channels, stimulus channels and auxillary channels
+    seldat  = startsWith(hdr.chantype, 'nirs');
+    selstim = strcmp(hdr.chantype, 'stimulus');
```

Comment:
this could also be trigger channels, right? I propose something like 

```
selstim = contains(hdr.chantype, 'stimulus') | contains(hdr.chantype, 'trigger');
```

to make it more general